packages/react/src/components/MeterChartCard/MeterChartCard.jsx

```diff
@@ -0,0 +1,144 @@
+
+import { CARD_SIZES } from '../../constants/LayoutConstants';
+import { CardPropTypes, MeterChartPropTypes } from '../../constants/CardPropTypes';
+import Card from '../Card/Card';
+import { MeterChart } from '../MeterChart';
+import { usePrevious } from '../../hooks/usePrevious';
+import { getUpdatedCardSize } from '../../utils/cardUtilityFunctions';
+
+const propTypes = {
+  ...CardPropTypes,
+  ...MeterChartPropTypes,
+};
+
+const defaultProps = {
+  size: CARD_SIZES.MEDIUMWIDE,
+  title: '',
+  content: {
+    peak: null,
+    meterTotal: 100,
+    meterUnit: 'Unit',
+    legendPosition: 'bottom',
+    color: {
+      pairing: {
+        option: 2,
+      },
+    },
+  },
+  footerContent: null,
+  locale: 'en',
+  values: [],
+  i18n: {
+    noDataLabel: 'No data available',
+  },
+};
+const MeterChartCard = ({
+  title,
+  content,
+  values,
+  size,
+  breakpoint,
+  footerContent,
+  locale,
+  isExpanded,
+  isLoading,
+  testId,
+  i18n,
+  ...others
+}) => {
+  const mergedI18n = { ...defaultProps.i18n, ...i18n };
+
+  const chartRef = useRef(null);
+  const contentWithDefaults = useMemo(() => defaultsDeep({}, content, defaultProps.content), [
+    content,
+  ]);
+
+  const isChartDataEmpty = isEmpty(values);
+
+  const options = useMemo(
+    () => ({
+      animations: false,
+      accessibility: true,
+      legend: {
+        position: contentWithDefaults.legendPosition,
+        enabled: values.length > 1,
+        truncation: contentWithDefaults.truncation,
+      },
+      containerResizable: true,
+      color: contentWithDefaults.colors,
+      meter: {
+        peak: contentWithDefaults.peak,
+        proportional: {
+          total: contentWithDefaults.meterTotal,
+          unit: contentWithDefaults.meterUnit,
+        },
+        status: {
+          ranges: [
+            ...(contentWithDefaults?.status?.success
+              ? [
+                  {
+                    range: contentWithDefaults.status.success,
+                    status: 'success',
+                  },
+                ]
+              : []),
+            ...(contentWithDefaults?.status?.warning
+              ? [
+                  {
+                    range: contentWithDefaults.status.warning,
+                    status: 'warning',
+                  },
+                ]
+              : []),
+            ...(contentWithDefaults?.status?.danger
+              ? [
+                  {
+                    range: contentWithDefaults.status.danger,
+                    status: 'danger',
+                  },
+                ]
+              : []),
+          ],
+        },
+      },
+      toolbar: {
+        enabled: false,
+      },
+    }),
+    [contentWithDefaults, values]
+  );
+  const newSize = useMemo(() => getUpdatedCardSize(size), [size]);
+  const previousChartData = usePrevious(values);
+
+  /** This is needed to update the chart when the lines and values change */
+  useEffect(() => {
```

Comment:
@cgirani Is it possible to bring these effects and size update functions into the base Card component? If we are going to add a prop type to Card in the future to dictate the type of chart we want the card to be able to handle all of these merges and size updates.

Reply:
it's implemented, having support for Meter/Sparkline/StackedArea [check the commit ](https://github.com/carbon-design-system/carbon-addons-iot-react/pull/3393/commits/1e6cef5e41abb88d95fac05f31a6ce005252d0b1)